web/accessibility.md

```diff
@@ -54,7 +54,26 @@ As more and more of our lives revolve around using the internet it is important
   - What did they do that worked well?
 
 4. Set up your screen reader:
-    Instructions for [Mac](https://support.apple.com/guide/mac-help/use-accessibility-features-mh35884/mac) and [PC](https://support.microsoft.com/en-us/windows/complete-guide-to-narrator-e4397a0d-ef4f-b386-d8ae-c172f109bdb1)
+    - Mac
+      - [VoiceOver](https://support.apple.com/guide/voiceover/welcome/mac)
+    - iOS Devices (iPad, iPhone)
+      - [VoiceOver](https://support.apple.com/guide/iphone/turn-on-and-practice-voiceover-iph3e2e415f/ios
+    - PC
```

Comment:
your going to have to do a lot more hand-holding here than handing a 2-month dev a 50 page technical guide. clearly they are just going to pick one, but can you limit this whole setup and test experience to 30 min or less? im sure you could find straight-forward directions already online somewhere.